Export Analysis Reports/Sdl.Community.ExportAnalysisReports/Helpers/Help.cs

```diff
@@ -38,7 +49,7 @@ public static string GetStudioProjectsPath()
 			return string.Empty;
 		}
 
-		public static Dictionary<string, LanguageDirection> LoadLanguageDirections(XmlDocument doc)
+		public Dictionary<string, LanguageDirection> LoadLanguageDirections(XmlDocument doc)
```

Comment:
If you add an item that's already in the dictionary it will throw an ArgumentException

Reply:
![image](https://user-images.githubusercontent.com/13149323/81153509-aafaa300-8f8b-11ea-887a-9d6f8558ea22.png)
I've added validation